src/platform_impl/linux/mod.rs

```diff
@@ -204,13 +226,15 @@ pub enum MonitorHandle {
 /// }
 /// ```
 /// The result can be converted to another enum by adding `; as AnotherEnum`
-macro_rules! x11_or_wayland {
+macro_rules! x11_or_wayland_or_drm {
```

Comment:
Especially if we add more backends in the future, it might be better to change the name of this (maybe `platform_specific!`)?